geotrek/common/templates/common/attachment_accessibility_list.html

```diff
@@ -4,7 +4,7 @@
     <strong>{% trans "Photos to illustrate information related to the accessibility of treks." %}</strong>
 </div>
 <div class="file-attachment-accessibility file-attachment-accessibility-detail">
-    <h3>{% trans "Attached files for accessibility" %}</h3>
+    <h3>{% trans "Photos accessibility " %}</h3>
```

Comment:
```suggestion
    <h3>{% trans "Pictures for accessibility" %}</h3>
```

take care of spaces in translations

Reply:
Ou sinon "Accessibility pictures"